src/components/HdfTwitter/HdfTwitter.jsx

```diff
@@ -33,150 +42,64 @@ export default function HdfTwitter() {
       id="Actualités"
       background={colors.hdfTwitter_background_color}
       text={colors.hdfTwitter_text_color}
+      display_actu1={translations.display_actu1}
+      display_actu2={translations.display_actu2}
+      display_actu3={translations.display_actu3}
+      display_actu4={translations.display_actu4}
+      display_actu5={translations.display_actu5}
```

Comment:
you need to delete this part

Reply:
Yes forgot that into the parent component :/